File: Miscelaneous/FuncionarioCLT.java
public class FuncionarioCLT extends Funcionario {
    private final double beneficio;

    public FuncionarioCLT(String nome, double salarioBase, double beneficio) {
        super(nome, salarioBase);
        this.beneficio = beneficio;
    }

    @Override public double calcularPagamento() { return salarioBase + beneficio; }
}

File: Miscelaneous/funcionario_clt.py
from funcionario import Funcionario


class FuncionarioCLT(Funcionario):
    def __init__(self, nome, salario_base, beneficio):
        super().__init__(nome, salario_base)
        self.beneficio = beneficio

    def calcular_pagamento(self):
        return self.salario_base + self.beneficio